thermo_nerf/scripts/train_eval_script.py

```diff
@@ -2,18 +2,25 @@
 from pathlib import Path
 
 import tyro
+
+from thermo_nerf.scripts.model_type import ModelType
+
 from nerfstudio.scripts.train import main
 
 from thermo_nerf.evaluator.evaluator import Evaluator
+from thermo_nerf.model_type import ModelType
```

Comment:
You are importing this file multiple time and they are not at the same place. Once if from the remove script one